packages/eslint-plugin-template/src/rules/no-negated-async.ts

```diff
@@ -25,7 +25,7 @@ export default createESLintRule<Options, MessageIds>({
     schema: [],
     messages: {
       noNegatedAsync:
-        'Async pipe results should not be negated. Use (observable | async) === (false || null || undefined) to check its value instead',
+        'Async pipe results should not be negated. Use (observable | async) === false, (observable | async) === null, or (observable | async) === undefined to check its value instead',
```

Comment:
While you are at it, could you please add the backticks?

```suggestion
        'Async pipe results should not be negated. Use `(observable | async) === false`, `(observable | async) === null` or `(observable | async) === undefined` to check its value instead',
```

Reply:
Good call, done!